tests/unit/combo/tests.html

```diff
@@ -5505,6 +5506,77 @@ <h2 id="qunit-userAgent"></h2>
                 equal(data.length, $listItems.length, "All items are visible");
 
             });
+
+            test(testId_92, function () {

```

Comment:
Now everything should be okay, thanks for pointing out that more tests should be implemented.